src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,25 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
   has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+
+  validates :project_id, presence: true
```

Comment:
belongs_to doesn't guarantee that?

Reply:
@coolo, in rails 5 yes, however:

```
initializers/new_framework_defaults.rb
20:Rails.application.config.active_record.belongs_to_required_by_default = false
```
setting it to true, do the trick